jcl/src/java.base/share/classes/jdk/internal/foreign/abi/UpcallLinker.java

```diff
@@ -74,44 +74,67 @@ public long entryPoint() {
-	 * @param session The SegmentScope of the upcall method handle
+	 * @param target the upcall method handle to the requested java method
+	 * @param methodType the MethodType of the upcall method handle
+	 * @param descriptor the FunctionDescriptor of the upcall method handle
+	 * @param session the SegmentScope of the upcall method handle
 	 * @return the native symbol
 	 */
-	public static MemorySegment make(MethodHandle target, MethodType mt, FunctionDescriptor cDesc, SegmentScope session)
+	public static MemorySegment make(MethodHandle target, MethodType methodType, FunctionDescriptor descriptor, SegmentScope session)
 	/*[ELSE] JAVA_SPEC_VERSION >= 20 */
 	/**
 	 * The method invoked via Clinker generates a native thunk to create
 	 * a native symbol that holds an entry point to the native function
 	 * intended for the requested java method in upcall.
 	 *
-	 * @param target The upcall method handle to the requested java method
-	 * @param mt The MethodType of the upcall method handle
-	 * @param cDesc The FunctionDescriptor of the upcall method handle
-	 * @param session The MemorySession of the upcall method handle
+	 * @param target the upcall method handle to the requested java method
+	 * @param methodType the MethodType of the upcall method handle
+	 * @param descriptor the FunctionDescriptor of the upcall method handle
+	 * @param session the MemorySession of the upcall method handle
 	 * @return the native symbol
 	 */
-	public static MemorySegment make(MethodHandle target, MethodType mt, FunctionDescriptor cDesc, MemorySession session)
+	public static MemorySegment make(MethodHandle target, MethodType methodType, FunctionDescriptor descriptor, MemorySession session)
 	/*[ENDIF] JAVA_SPEC_VERSION >= 20 */
 	{
-		UpcallLinker upcallLinker = new UpcallLinker(target, mt, cDesc, session);
+		UpcallLinker upcallLinker = new UpcallLinker(target, methodType, descriptor, session);
 		return UpcallStubs.makeUpcall(upcallLinker.entryPoint(), session);
 	}
 
 	/*[IF JAVA_SPEC_VERSION >= 21]*/
 	/**
-	 * A stub method to be implemented.
+	 * A stub method instended in OpenJDK to support compilation.
 	 *
-	 * @param targetType
-	 * @param abi
-	 * @param callingSequence
-	 * @return
+	 * Note:
+	 * SharedUtils.arrangeUpcallHelper() (totally featured with the
+	 * signatures intended for OpenJDK) calls this method which
+	 * is specific to the upcall specific code implemented in
+	 * OpenJDK. To work around this case during the compilation,
+	 * this stub method serves as a placeholder to minimize the
+	 * changes in OpenJDK.
+	 *
+	 * @param methodType the MethodType of the upcall method handle
+	 * @param abi the descriptor of the Application Binary Interface(ABI) on a given platform
+	 * @param callingSequence the calling sequence converted by CallArranger from a C FunctionDescriptor
+	 * @return a factory instance that wraps up the upcall specific code
 	 * @throws UnsupportedOperationException
```

Comment:
Please insert a space before `(ABI)` and complete the `@throws` clause.

Reply:
Fixed.